docs/apache-airflow/extra-packages-ref.rst

```diff
@@ -68,6 +68,8 @@ python dependencies for the provided package.
 +---------------------+-----------------------------------------------------+----------------------------------------------------------------------------+
 | sentry              | ``pip install 'apache-airflow[sentry]'``            | Sentry service for application logging and monitoring                      |
 +---------------------+-----------------------------------------------------+----------------------------------------------------------------------------+
+| s3fs                | ``pip install 'apache-airflow[s3fs]'``              | Support for S3 as Airflow FS                                               |
```

Comment:
```suggestion
| s3fs                | ``pip install 'apache-airflow[s3fs]'``              | Support for S3 as Airflow Store                                            |
```

Reply:
I wonder if it is better to try integrating this into the amazon provider.

Reply:
I agree, I was struggling a bit here finding my way around. How to do that?

Reply:
The `amazon` extra actually just pulls in `apache-airflow-providers-amazon` so this would go into the provider.yaml config.

Reply:
(Sorry for the noob questions ;-) ) But how do get the amazon-extras installed? `s3fs` brings in `aiobotocore` which is treated as a special case due to its hard dependency on a particular boto3 version.

Reply:
So it would simply install (and downgrade botocore/boto3) by `pip install apache-airflow-providers-amazon[aiobotocore]`

Reply:
Just verifying @potiuk should `s3fs` now be kept in `setup.py` or should I move it out again?

Reply:
keep it. Pre-commit will tell you if it's OK.